packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -565,16 +558,36 @@ export default {
     }
 }
 
-.c-cookieBanner-cta {
+.c-cookieBanner-cta-container {
     padding-block: f.spacing(d) f.spacing(a);
     padding-inline: f.spacing(e);
 
+    .c-cookieBanner-cta {
+        width: 100%;
```

Comment:
Please can you use logical properties instead, i.e., inline-size, margin-block-end, margin-inline-start.

Reply:
Fixed. https://github.com/justeattakeaway/fozzie-components/pull/2281 For some reason the branch update didn't push to the right place so i have a new pr.